src/Form/selectors-css.js

```diff
@@ -9,17 +9,17 @@ button:not([role=switch]):not([role=link]),
 [role=button]`
 
 const email = `
-input:not([type])[name*=mail i],
-input[type=""][name*=mail i],
-input[type=text][name*=mail i],
-input:not([type])[placeholder*=mail i]:not([placeholder*=search i]),
-input[type=text][placeholder*=mail i]:not([placeholder*=search i]),
-input[type=""][placeholder*=mail i]:not([placeholder*=search i]),
-input:not([type])[placeholder*=mail i]:not([placeholder*=search i]),
+input:not([type])[name*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
+input[type=""][name*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
+input[type=text][name*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
+input:not([type])[placeholder*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
+input[type=text][placeholder*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
+input[type=""][placeholder*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
+input:not([type])[placeholder*=mail i]:not([placeholder*=search i]):not([placeholder*=filter i]):not([placeholder*=subject i]),
```

Comment:
Again, removing `placeholder` and `subject` from the match. Ultimately we may want to move away from these complex selectors and have a more robust logic with regexes. Too risky now.

Reply:
agreed - a lot of the complexity in `matchers.js` is around supporting the 3 layers. It's something we can work to reduce